contracts/external/cw-verifier-middleware/src/utils.rs

```diff
@@ -0,0 +1,40 @@
+use cosmwasm_std::{to_binary, Api, HexBinary};
+use secp256k1::{hashes::hex::ToHex, rand::rngs::OsRng, Message, Secp256k1};
+use sha2::{Digest, Sha256};
+
+use crate::{
+    msg::{Payload, WrappedMessage},
+    verify::{get_sign_doc, pk_to_addr},
+};
+
+// signs a given payload and returns the wrapped message
+pub fn get_wrapped_msg(api: &dyn Api, payload: Payload) -> WrappedMessage {
```

Comment:
```suggestion
// signs a given payload and returns the wrapped message
#[cfg(test)]
pub fn get_wrapped_msg(api: &dyn Api, payload: Payload) -> WrappedMessage {
```

was exploding my brain before i realized this